sodasql/scan/test.py

```diff
@@ -32,4 +31,5 @@ def evaluate(self, test_variables: dict, group_values: Optional[dict] = None):
             logging.debug(str(test_result))
             return test_result
         except Exception as e:
+            logging.debug(e)
```

Comment:
I think this will not log the stack trace, right? This is fine if your intention is only to get the error message, but maybe there is a way to debug only (less noise) with exception stack trace

Reply:
Actually I added this while testing and forgot to remove it.